config/rules_of_origin_tab.yml

```diff
@@ -0,0 +1,10 @@
+---
+rules_of_origin_tab:
```

Comment:
Just my 2c but I'm wondering whether we should put this in `config/locales/*` since it feels like some text that we might want to translate in future?

Reply:
I think you can then do:

```ruby
I18n.t("rules_of_origin_tab.preferential_treatment.#{country_id}", default: '')
```

Reply:
I actually think you might be able to do:

```ruby
I18n.t("rules_of_origin_tab.preferential_treatment.#{country_id}_html", default: '') # spot the _html
```

Reply:
https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

Reply:
good point,
I used a different locale file instead of putting all inside `en.yml` because the preferential treatments are pretty long chunk of HTML.

Reply:
So I actually think that's okay for reasons I mention, here: https://github.com/trade-tariff/trade-tariff-frontend/pull/839/files#r883507520